modules/standard/BigInteger.chpl

```diff
@@ -1525,53 +1525,23 @@ module BigInteger {
        in an integer.  When ``numer/denom`` does not produce an integer, this
        method may produce incorrect results.
 
-    :arg numer: numerator
+    Utilizes the GMP function `mpz_divexact
+    <https://gmplib.org/manual/Integer-Division>`_.
 
+    :arg numer: numerator
     :type numer: :record:`bigint`
 
     :arg denom: denominator
-
     :type denom: :record:`bigint` or ``integral``
   */
-  @deprecated(notes="bigint.divexact method is deprecated - please use the standalone function :proc:`~BigInteger.divexact`")
-  proc bigint.divexact(const ref numer: bigint, denom: integral) {
-    BigInteger.divexact(this, numer, denom);
-  }
-
-  // divisible_p
-  /*
-    .. warning::
-
-       bigint.divisible_p is deprecated, use bigint.isDivisible instead
-  */
-  @deprecated
-  ("bigint.divisible_p is deprecated, use bigint.isDivisible instead")
-  proc bigint.divisible_p(const ref d: bigint) : int {
-    return this.isDivisible(d);
-  }
-  /*
-    .. warning::
-
-       bigint.divisible_p is deprecated, use bigint.isDivisible instead
-  */
-  @deprecated
-  ("bigint.divisible_p is deprecated, use bigint.isDivisible instead")
-  proc bigint.divisible_p(d: int) : int {
-    return this.isDivisible(d);
-  }
-  /*
-    .. warning::
+  @deprecated(notes="bigint.divexact method is deprecated - please use the standalone function :proc:`~BigInteger.divExact`")
+  proc bigint.divexact(const ref numer: bigint, denom: integral)
+    do BigInteger.divExact(this, numer, denom);
 
-       bigint.divisible_p is deprecated, use bigint.isDivisible instead
-  */
-  @deprecated
-  ("bigint.divisible_p is deprecated, use bigint.isDivisible instead")
-  proc bigint.divisible_p(d: uint) : int {
-    return this.isDivisible(d);
-  }
 
   // divisible_p
   // documented in uint version
+  @chpldoc.nodoc
```

Comment:
Is there a reason these `isDivisible` functions are being `nodoc`ed? I don't see a mention of that in the PR message. I see the comment about it being documented in the `uint` version, but I'm not sure that means we want to remove it from the documentation entirely, since it could look like only `uint` was working, unless I'm missing something.

Reply:
I thought I removed all those, removing now!